server/core/config/base.py

```
@@ -14,6 +14,10 @@
 from pathlib import Path
 from dotenv import load_dotenv
 
+# third party imports
+from .celery_config import (CELERY_BROKER_URL, CELERY_RESULT_BACKEND, CELERY_ACCEPT_CONTENT, CELERY_TASK_SERIALIZER, CELERY_RESULT_SERIALIZER, CELERY_TIMEZONE)
```

Ruff (0.8.0)

`.celery_config.CELERY_BROKER_URL` imported but unused

Remove unused import

(F401)